src/pages/CustomerInvoicePage.js

```diff
@@ -627,7 +648,7 @@ CustomerInvoicePage.propTypes = {
   database: PropTypes.object.isRequired,
   genericTablePageStyles: PropTypes.object.isRequired,
   runWithLoadingIndicator: PropTypes.func.isRequired,
-  topRoute: PropTypes.bool.isRequired,
+
```

Comment:
Remove blank line.
That said, I only left this here cause it was part of a feature: Refresh a page table if navigating back to it.

Reply:
1f070fedd9c8e44b55a3b1d298669f89383e0e89